bin/metrics-rabbitmq-overview.rb

```diff
@@ -53,9 +54,9 @@ class RabbitMQMetrics < Sensu::Plugin::Metric::CLI::Graphite
          proc: proc(&:to_i),
          default: 15_672
 
-  option :user,
+  option :username,
```

Comment:
I like making it consistent but this makes it a breaking change. This will need to be called out in the CHANGELOG.